docs/_static/auth/iam.yaml

```diff
@@ -149,6 +153,52 @@ paths:
                 "$ref": "#/components/schemas/PresentationDefinition"
         "default":
           $ref: '../common/error_response.yaml'
+  "/iam/{id}/response":
```

Comment:
is `/response` specific enough? Or should it be something like `/response_direct_post` or `/authz_response`

Reply:
There's a description on it. The other oauth2/openid4vp specific endpoints are: `token` and `authorize`, so a simple name is in line with the others.